src/pages/DriverDashboard/DriverDashboard.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Flex } from '@chakra-ui/react';
+import './DriverDashboard.module.css';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './DriverDashboard.module.css'.